.devcontainer/post_create_command.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+npm install
+cd frontend/ && npm install
+cd ..
```

Comment:
Just a note I like what we did for our [lambda_dependencies](https://github.com/awslabs/mlspace/blob/main/lambda_dependencies/jwt/create.sh#L6) where we use `pushd new/path` and then `popd` to get back to the original directory.  It doesn't matter as much here because we're only going one directory deep so we're not doing something like `../../..` but sharing for future knowledge where it makes more sense.